cypher/cypher-docs/src/docs/dev/deprecations-additions-and-compatibility.asciidoc

```diff
@@ -440,6 +463,28 @@ CREATE INDEX ...
 a|
 The default index type is changed from B-tree to range index.
 
+a|
+label:functionality[]
+label:updated[]
+[source, cypher, role="noheader"]
+----
+SHOW INDEXES
+----
+a|
+A new column is added, `owningConstraint`. The column is returned by default.
+This column returns the name of the constraint the index is associated with, or `null` if it is not associated with any constraint.
```

Comment:
```suggestion
It will list the name of the constraint that the index is associated with or `null`, in case it is not associated with any.
```

Reply:
I feel like the sentence ends very abruptly on the `any`, but that might just be me?
 ```suggestion
It will list the name of the constraint that the index is associated with or `null`, in case it is not associated with any constraint.
```

Reply:
I think in terms of grammar it's fine to end with "any", but we can go for your suggestion with no problems! It's also correct and we can avoid any other readers finding it weird. Thanks!